### PR TITLE
Ft api post event 152880657

### PR DESCRIPTION
**Pull request by seunzone**
**What this PR do**
Created the POST request for the events of the project.
Recent commits includes PUT and DELETE routes for events & also test has be written following TDD best  practices convention. 
The ID of this feature on pivotal tracker is 152880657

**Running the App on your local machine**
To install dependencies: `npm install`
To run test: `npm run test`
To run the application: `npm start`

To test the Route, use postman and send a post request to:
localhost:3000/api/v1/event

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "event-manager",
+  "version": "1.0.0",
+  "description": "This App allows client post events online and also grants admin access level to the site administrators to smoothly run the the site.",
+  "main": "server/app.js",
+  "scripts": {
+    "start": "babel-watch server/app.js",
+    "babel-node": "babel-node --",
```
Comment (ghost) [inline package.json]: What would you be using this script for?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "build": "./node_modules/.bin/babel server -d build",
+    "dist": "./node_modules/.bin/babel server -d dist --presets=babili"
```
Comment (ghost) [inline package.json]: What is this also used for??

File: server/controllers/controllerEvent.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+   * @param {any} req
+   * @param {any} res
+   * @returns {json} adds an event
+   * @memberof Event
+   */
+  addEvent(req, res){
+      const{
+        userId, title, date, time, venue, details, centerId
+      } = req.body;
+      if (!userId){
```
Comment (ghost) [inline server/controllers/controllerEvent.js]: Rather this long if statements, use a middleware to validate your request body before passing it to the addEvent function

File: package.json (hunk 1 above)
Comment (seunzone) [reply package.json]: babel-watch works just like nodemon, it watches for changes in my code after I save it and restarts the server

File: package.json (hunk 2 above)
Comment (seunzone) [reply package.json]: Its not necessary, initially I was trying to play around running my code from a build folder, I'll adjust that

File: server/controllers/controllerEvent.js (hunk 3 above)
Comment (seunzone) [reply server/controllers/controllerEvent.js]: ok, I'll do some research on that.

Comment (seunzone) [reply server/controllers/controllerEvent.js]: Will fix that in a while

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json]: I am talking of the `babel-node` script

Comment (seunzone) [reply package.json]: ohh, I see it, some real mistake there. Would take that off

**Pull request merged**